### PR TITLE
add ekscloudwatch

### PR DESCRIPTION
**Pull request by dza89**
**What type of PR is this?**
Enhancement

/kind feature

**Any specific area of the project related to this PR?**

/area falco-chart

**What this PR does / why we need it**:

Includes ekscloudwatch as a deployment.

**Which issue(s) this PR fixes**:
Approaches native functionality on eks.

- [ ] Chart Version bumped
- [X] Variables are documented in the README.md
- [x] CHANGELOG.md updated

### CODE REVIEW

File: falco/README.md [hunk 1]
```diff
@@ -139,6 +139,19 @@ The following table lists the configurable parameters of the Falco chart and the
...
+| `eksCloudwatch.image.registry`                                | The image registry to pull from                                                                                    | `docker.io`                                                                                                                               |
+| `eksCloudwatch.image.repository`                              | The image repository to pull from                                                                                  | `falcosecurity/falco`                                                                                                                     |
+| `eksCloudwatch.image.tag`                                     | The image tag to pull                                                                                              | `0.21.0`                                                                                                                                  |
```
Comment (leogr) [inline falco/README.md:146]: It seems to me that default values are different:

```
image:
    registry: docker.io
    repository: sysdiglabs/k8sauditlogforwarder
    tag: ekscloudwatch-0.2
```

Could you fix that please?

Comment (dza89) [reply falco/README.md]: Done

Comment (leogr) [reply falco/README.md]: I still see:

```
| `eksCloudwatch.image.repository`                              | The image repository to pull from                                                                                  | `falcosecurity/falco`                                                                                                                     |
| `eksCloudwatch.image.tag`                                     | The image tag to pull                                                                                              | `0.2.0`
```

**Comment by dza89** [comment]
@leogr 
Thanks. Sorry fixed.

My input: ekscloudwatch is an extension of Falco, and cannot live on it's own, so should be it in the same helm chart.

I also don't understand why falco-exporter is in a different chart and not a side container within the falco helm chart for the same reason. Deploying it seperately introduces an extra (unneccesary) risk. (Volume mounting a hostfilepath)

**Comment by leogr** [comment]
Thank you, @dza89 !

There're several reasons why falco-exporter (and others, we will soon get falcosidekick here too) are distinct charts. IMHO, it's mainly a matter of maintainability and modularity. For example, the Falco chart is already too cumbersome to get the full official support (as per [our convention](https://github.com/falcosecurity/evolution#official-support)) because of all embedded integrations that are hard to test and maintain. But, there're cons too. As always, it's a trade-off.

Moreover, the community could raise other concerns about this integration. AFAIK `sysdiglabs/k8sauditlogforwarder` is not maintained by the Falco project, does not seem to be documented, and I haven't found the source code of that image. Perhaps, do you know where I can see the sources?

That being said, I can understand your point, and I will not disagree if maintainers want to merge this. 
I'm just trying to be a bit scrupulous before including a new thing in the chart :)

**Comment by dza89** [comment]
Ah ok, I thought sysdiglabs and falcosecurity were practically the same thing :)
Personally I don't mind a big helm chart (until it becomes prometheus-operator)
And you do need this to have falco fully functioning on EKS. So perhaps a cloudprovider piece which you refrain from testing. (?)

Source:
https://github.com/sysdiglabs/ekscloudwatch

**Review by leodido — COMMENTED** [review]
IMHO, this is great work.

Anyways, it needs to be brought to our Community Calls to decide together whether we do want it here, or in `evolution` (for example). Basically, to decide how we want to handle such situations.

My 0.02 cents

**Comment by leogr** [comment]
Hey @dza89 

I have seen your commit https://github.com/falcosecurity/charts/pull/63/commits/eb523cbb308d57838fa661180ca66b8fcd346431 that introduces the grafana dashboard for falco-exporter, anyway I believe this feature should be introduced in a different PR and not here (since it affects a different chart and is not directly related to ekscloudwatch).

**Pull request closed without merging**